File: Week_7/problem_5-Lucky_array.py
# https://codeforces.com/group/MWSDmqGsZm/contest/219774/problem/J

n = int(input())
ar = list(map(int, input().split()))  # Read the array of integers

# Find the minimum element in the array
minimum_element = min(ar)

# Count how many times the minimum element appears
count = ar.count(minimum_element)

# Check if the count of the minimum element is odd or even
if count % 2 != 0:
    print("Lucky")
else:
    print("Unlucky")

File: Week_7/problem_5-Lucky_array.c
// https://codeforces.com/group/MWSDmqGsZm/contest/219774/problem/J

#include <stdio.h>
#include <limits.h>
#include <string.h>
#include <math.h>
#include <stdlib.h>

int main()
{
    int n;
    scanf("%d", &n);
    int ar[n];

    for (int i = 0; i < n; i++)
    {
        scanf("%d", &ar[i]);
    }

    int minimum_element = INT_MAX;
    for (int i = 0; i < n; i++)
    {
        if (ar[i] < minimum_element)
            minimum_element = ar[i];
    }
    // printf("%d\n", minimum_element);

    int count = 0;
    for (int i = 0; i < n; i++)
    {
        if (ar[i] == minimum_element)
            count++;
    }
    // printf("%d\n", count);

    if (count % 2 != 0)
    {
        printf("Lucky");
    }
    else
    {
        printf("Unlucky");
    }

    return 0;
}
